app/main.py

```diff
@@ -1,5 +1,55 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
```

Comment:
coords could have both types of values, use annotation like this. The same in all other similar cases

```suggestion
    def __init__(self, name: str, weight: int, coords: list | None= None) -> None:
```